scripts/runXBRL21Tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Run XBRL 2.1 Conformance Suite tests
+
+TESTCASESROOT=/users/hermf/Documents/mvsl/projects/XBRL.org/conformance-svn/trunk
+rm -f /users/hermf/temp/XBRL21-conf-*
+OUTPUTLOGFILE=/users/hermf/temp/XBRL21-conf-log.txt
+OUTPUTERRFILE=/users/hermf/temp/XBRL21-conf-err.txt
+OUTPUTCSVFILE=/users/hermf/temp/XBRL21-conf-report.xlsx
+TESTCASESINDEXFILE=$TESTCASESROOT/xbrl.xml
+ARELLEDIR=/users/hermf/Documents/mvsl/projects/arelle/arelleproject/src
```

Comment:
Nit: seems like your computer specific for a bunch of these values

Reply:
All the test scripts under "scripts" have a similar setup, there should be a way to improve  them for CI as well as stand alone operation.